lib/open_project/static/links.rb

```diff
@@ -251,6 +251,9 @@ def static_links
               href: "https://www.openproject.org/docs/user-guide/time-and-costs/progress-tracking/"
             },
             enterprise_docs: {
+              automatic_subject_generation: {
+                href: "https://www.openproject.org/docs/todo/do-not-merge"
```

Comment:
Can we already know/guess the correct URL this will have in the future? Otherwise we'll need to remember to revisit this link and fix it before public release.

Reply:
This needs to be discussed with Maya. Maybe she can predict it, if not, we need to put a placeholder.

Reply:
Aligned with @MayaBerd and added a small placeholder into the docs already and pointed to the correct place.

Note: We should now not merge this branch before todays feature freeze (so that the docs are not updated before 15.3).